pycloudflare/models.py

```diff
@@ -197,6 +197,11 @@ def purge_cache(self, files=None, tags=None):
     def get_ssl_verification_info(self):
         return self._service.get_ssl_verification_info(self.id)
 
+    def re_verify(self):
+        for state in (False, True):
```

Comment:
This looks like something that should have an explanatory comment.  The "'Toggle Universal SSL off and on, to trigger re-verification'" (the help for the management commands) seems good enough.

Reply:
Yeah. Also document the side-effect that SSL gets enabled.